extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -172,6 +174,10 @@ public static function onOasisSkinAssetGroupsBlocking( &$jsAssets ) {
 			$jsAssets[] = self::ASSET_GROUP_ADENGINE_RUBICON_FASTLANE;
 		}
 
+		if ( AnalyticsProviderRubiconVulcan::isEnabled() ) {
+			$jsAssets[] = self::ASSET_GROUP_ADENGINE_RUBICON_VULCAN;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)